__tests__/github-mention-test.js

```diff
@@ -19,20 +20,60 @@ describe('Github Mention', function() {
   // If you are working on the algorithm itself, it is useful to be able to run
   // the complete flow that downloads the diff and subsequent blames. Since
   // doing http requests is unreliable in tests, it is disabled by default.
-  xit('CompleteFlow', function() {
-    mentionBot.enableCachingForDebugging = true;
-    var prs = [3238];
-    prs.forEach(function(i) {
-      console.log(
-        i,
-        mentionBot.guessOwnersForPullRequest(
-          'https://github.com/facebook/react-native',
+  xdescribe('CompleteFlow', function() {
+    it('Gets correct users with no config options', function() {
+      mentionBot.enableCachingForDebugging = true;
+      var prs = [3238];
+      prs.forEach(function(i) {
+        console.log(
           i,
-          'mention-bot',
-          'master',
-          {} //config
-        )
+          mentionBot.guessOwnersForPullRequest(
+            'https://github.com/facebook/react-native',
+            i,
+            'mention-bot',
+            'master',
+            {
+              userBlacklist: []
+            } //config
```

Comment:
the `// config` was there because it is unclear what `{}` means. But `{userBlacklist: []}` is pretty explicit so you can remove that comment :)